Reference/conformance/conformance.pb.swift

```diff
@@ -163,7 +163,13 @@ struct Conformance_ConformanceRequest: SwiftProtobuf.Message {
   /// other serializer methods are defined in the SwiftProtobuf library. See the
   /// `Message` and `Message+*Additions` files.
   func traverse<V: SwiftProtobuf.Visitor>(visitor: inout V) throws {
-    try self.payload?.traverse(visitor: &visitor)
+    switch self.payload {
```

Comment:
`nil` is the same as `.none` since we're switching on an `Optional`. You can also write `case nil`, in fact. Similarly, the pattern `.foo?` is equivalent to `.some(.foo)`.